python/lsst/ip/diffim/dipoleFitTask.py

```diff
@@ -791,12 +791,12 @@ def fitDipole(self, source, tol=1e-7, rel_weight=0.1,
             Weighting of posImage/negImage relative to the diffim in the fit
         fitBackground : `int`, {0, 1, 2}, optional
             How to fit linear background gradient in posImage/negImage
-            - 0: do not fit background at all
-            - 1 (default): pre-fit the background using linear least squares and then do not fit it as part
-                 of the dipole fitting optimization
-            - 2: pre-fit the background using linear least squares (as in 1), and use the parameter
-                 estimates from that fit as starting parameters for an integrated "re-fit" of the background
-                 as part of the overall dipole fitting optimization.
+                - 0: do not fit background at all
+                - 1 (default): pre-fit the background using linear least squares and then do not fit it
+                  as part of the dipole fitting optimization
+                - 2: pre-fit the background using linear least squares (as in 1), and use the parameter
+                  estimates from that fit as starting parameters for an integrated "re-fit" of the
+                  background as part of the overall dipole fitting optimization.
```

Comment:
Can you please explain why these small docstring formatting changes are part of this PR? (Here as well as imagePsfMatch.py and zogy.py.)

Reply:
These are just typos that break the docstring rendering. According to John, minor changes that don't deserve their standalone ticket can be simply added to issues as they emerge.

Reply:
Without this allowance the overhead to doing minor cleanups as you find them whilst working on something else would be too high. It's always more efficient to fix the little docstring problem whilst you notice it than have to file a ticket and delay the fix. So long as the fixups are on distinct commits (as here) then all is fine.